### ISSUE TITLE
compile problem of freeradius-server-3.x

### ISSUE THREAD

**Issue by NolanWoo** [FreeRADIUS/freeradius-server#2245]
# Issue type
compile problem of freeradius-server-3.x

***

**REMOVE THOSE WHICH DO NOT APPLY**

See [here](https://github.com/FreeRADIUS/freeradius-server/blob/v4.0.x/doc/bugs.md) for debugging instructions and how to obtain backtraces.

# Defect
## How to reproduce the issue

when I use "make"  to compile, some compile problem occurs. I have never changed any configurations, before I make, I use "./configure" to config withou any arguments.
 
## Output of ``[radiusd|freeradius] -X`` showing issue occurring

```text
CC src/lib/cbuff.c
CC src/lib/cursor.c
CC src/lib/debug.c
CC src/lib/dict.c
CC src/lib/filters.c
CC src/lib/hash.c
CC src/lib/hmacmd5.c
CC src/lib/hmacsha1.c
CC src/lib/isaac.c
CC src/lib/log.c
CC src/lib/misc.c
CC src/lib/missing.c
CC src/lib/md4.c
CC src/lib/md5.c
CC src/lib/net.c
src/lib/net.c: In function ‘fr_link_layer_supported’:
src/lib/net.c:35:7: error: ‘DLT_EN10MB’ undeclared (first use in this function)
  case DLT_EN10MB:
       ^
src/lib/net.c:35:7: note: each undeclared identifier is reported only once for each function it appears in
src/lib/net.c:36:7: error: ‘DLT_RAW’ undeclared (first use in this function)
  case DLT_RAW:
       ^
src/lib/net.c:37:7: error: ‘DLT_NULL’ undeclared (first use in this function)
  case DLT_NULL:
       ^
src/lib/net.c:38:7: error: ‘DLT_LOOP’ undeclared (first use in this function)
  case DLT_LOOP:
       ^
src/lib/net.c:42:7: error: ‘DLT_PFLOG’ undeclared (first use in this function)
  case DLT_PFLOG:
       ^
src/lib/net.c: In function ‘fr_link_layer_offset’:
src/lib/net.c:71:7: error: ‘DLT_RAW’ undeclared (first use in this function)
  case DLT_RAW:
       ^
src/lib/net.c:74:7: error: ‘DLT_NULL’ undeclared (first use in this function)
  case DLT_NULL:
       ^
src/lib/net.c:75:7: error: ‘DLT_LOOP’ undeclared (first use in this function)
  case DLT_LOOP:
       ^
src/lib/net.c:85:7: error: ‘DLT_EN10MB’ undeclared (first use in this function)
  case DLT_EN10MB:
       ^
src/lib/net.c:133:7: error: ‘DLT_PFLOG’ undeclared (first use in this function)
  case DLT_PFLOG:
       ^
src/lib/net.c: At top level:
cc1: warning: unrecognized command line option "-Wno-date-time" [enabled by default]
make: *** [build/objs/src/lib/net.lo] Error 1

```
## Full backtrace from LLDB or GDB

```text
COPY/PASTE OUTPUT HERE (WITHIN BACKTICKS).  NO PASTEBIN (ET AL) LINKS!
```

**Comment by arr2036** [comment]
Please provide the contents of config.log

**Comment by NolanWoo** [comment]
config.log

**Comment by NolanWoo** [comment]
configure:3539: $? = 0
configure:3528: g++ -v >&5
Using built-in specs.
COLLECT_GCC=g++
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/4.8/lto-wrapper
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Ubuntu 4.8.4-2ubuntu1~14.04.3' --with-bugurl=file:///usr/share/doc/gcc-4.8/README.Bugs --enable-languages=c,c++,java,go,d,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.8 --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.8 --libdir=/usr/lib --enable-nls --with-sysroot=/ --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --enable-gnu-unique-object --disable-libmudflap --enable-plugin --with-system-zlib --disable-browser-plugin --enable-java-awt=gtk --enable-gtk-cairo --with-java-home=/usr/lib/jvm/java-1.5.0-gcj-4.8-amd64/jre --enable-java-home --with-jvm-root-dir=/usr/lib/jvm/java-1.5.0-gcj-4.8-amd64 --with-jvm-jar-dir=/usr/lib/jvm-exports/java-1.5.0-gcj-4.8-amd64 --with-arch-directory=amd64 --with-ecj-jar=/usr/share/java/eclipse-ecj.jar --enable-objc-gc --enable-multiarch --disable-werror --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
Thread model: posix
gcc version 4.8.4 (Ubuntu 4.8.4-2ubuntu1~14.04.3) 
configure:3539: $? = 0
configure:3528: g++ -V >&5
g++: error: unrecognized command line option '-V'
g++: fatal error: no input files
compilation terminated.
configure:3539: $? = 4
configure:3528: g++ -qversion >&5
g++: error: unrecognized command line option '-qversion'
g++: fatal error: no input files
compilation terminated.
configure:3539: $? = 4
configure:3543: checking whether we are using the GNU C++ compiler
configure:3562: g++ -c   conftest.cpp >&5
configure:3562: $? = 0
configure:3571: result: yes
configure:3580: checking whether g++ accepts -g
configure:3600: g++ -c -g  conftest.cpp >&5
configure:3600: $? = 0
configure:3641: result: yes
configure:3671: checking how to run the C preprocessor
configure:3702: gcc -E  conftest.c
configure:3702: $? = 0
configure:3716: gcc -E  conftest.c
conftest.c:11:28: fatal error: ac_nonexistent.h: No such file or directory
 #include <ac_nonexistent.h>
                            ^
compilation terminated.
configure:3716: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "freeradius"
| #define PACKAGE_TARNAME "freeradius"
| #define PACKAGE_VERSION "$Id: 0c59a3c8a9d5583ab592b011bb88ef20d075820c $"
| #define PACKAGE_STRING "freeradius $Id: 0c59a3c8a9d5583ab592b011bb88ef20d075820c $"
| #define PACKAGE_BUGREPORT "http://bugs.freeradius.org"
| #define PACKAGE_URL "http://www.freeradius.org"
| #define RADIUSD_VERSION 030018
| #define RADIUSD_VERSION_STRING "3.0.18"
| /* end confdefs.h.  */
| #include <ac_nonexistent.h>
configure:3741: result: gcc -E
configure:3761: gcc -E  conftest.c
configure:3761: $? = 0
configure:3775: gcc -E  conftest.c
conftest.c:11:28: fatal error: ac_nonexistent.h: No such file or directory
 #include <ac_nonexistent.h>
                            ^
compilation terminated.
configure:3775: $? = 1
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "freeradius"
| #define PACKAGE_TARNAME "freeradius"
| #define PACKAGE_VERSION "$Id: 0c59a3c8a9d5583ab592b011bb88ef20d075820c $"
| #define PACKAGE_STRING "freeradius $Id: 0c59a3c8a9d5583ab592b011bb88ef20d075820c $"
| #define PACKAGE_BUGREPORT "http://bugs.freeradius.org"
| #define PACKAGE_URL "http://www.freeradius.org"
| #define RADIUSD_VERSION 030018
| #define RADIUSD_VERSION_STRING "3.0.18"
| /* end confdefs.h.  */
| #include <ac_nonexistent.h>
configure:3804: checking for grep that handles long lines and -e
configure:3862: result: /bin/grep
configure:3867: checking for egrep
configure:3929: result: /bin/grep -E
configure:3934: checking for ANSI C header files
configure:3954: gcc -c -g -O2  conftest.c >&5
configure:3954: $? = 0
configure:4027: gcc -o conftest -g -O2   conftest.c  >&5
configure:4027: $? = 0
configure:4027: ./conftest
configure:4027: $? = 0
configure:4038: result: yes
configure:4051: checking for sys/types.h
configure:4051: gcc -c -g -O2  conftest.c >&5
configure:4051: $? = 0
configure:4051: result: yes
configure:4051: checking for sys/stat.h
configure:4051: gcc -c -g -O2  conftest.c >&5
configure:4051: $? = 0
configure:4051: result: yes
configure:4051: checking for stdlib.h
configure:4051: gcc -c -g -O2  conftest.c >&5
configure:4051: $? = 0
configure:4051: result: yes
configure:4051: checking for string.h
configure:4051: gcc -c -g -O2  conftest.c >&5
configure:4051: $? = 0
configure:4051: result: yes
configure:4051: checking for memory.h
configure:4051: gcc -c -g -O2  conftest.c >&5
configure:4051: $? = 0
configure:4051: result: yes
configure:4051: checking for strings.h
configure:4051: gcc -c -g -O2  conftest.c >&5
configure:4051: $? = 0
configure:4051: result: yes
configure:4051: checking for inttypes.h
configure:4051: gcc -c -g -O2  conftest.c >&5
configure:4051: $? = 0
configure:4051: result: yes
configure:4051: checking for stdint.h
configure:4051: gcc -c -g -O2  conftest.c >&5
configure:4051: $? = 0
configure:4051: result: yes
configure:4051: checking for unistd.h
configure:4051: gcc -c -g -O2  conftest.c >&5
configure:4051: $? = 0
configure:4051: result: yes
configure:4064: checking minix/config.h usability
configure:4064: gcc -c -g -O2  conftest.c >&5
conftest.c:54:26: fatal error: minix/config.h: No such file or directory
 #include <minix/config.h>
                          ^
compilation terminated.

**Comment by arr2036** [comment]
None of this is useful.  If you are unable to use pastebin, please copy/paste the section around where libpcap is being checked for.

**Comment by NolanWoo** [comment]
I‘m sorry I have tried, but I have no right to upload files beyond a certain size( even I use pastebin ) because of company regulation.
 In config.log every "HAVE_LIBPCAP" is 1

**Comment by NolanWoo** [comment]
configure:6782: result: no
configure:6847: checking for pcap_open_live in -lpcap in /opt/docker/aufs/diff/1458d7dbf760601e9276866f8d31c399bb56268e8bf931a847893e315a749abb/pdt-toolchains/8/amd64/sysroot/usr/lib/i386-linux-gnu
configure:6862: gcc -o conftest -g -O2 -Wall -std=c99 -D_GNU_SOURCE -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS -L/opt/docker/aufs/diff/1458d7dbf760601e9276866f8d31c399bb56268e8bf931a847893e315a749abb/pdt-toolchains/8/amd64/sysroot/usr/lib/i386-linux-gnu -Wl,-rpath,/opt/docker/aufs/diff/1458d7dbf760601e9276866f8d31c399bb56268e8bf931a847893e315a749abb/pdt-toolchains/8/amd64/sysroot/usr/lib/i386-linux-gnu   conftest.c -lpcap -lnsl -lresolv -ldl -lpthread  >&5
/usr/bin/ld: skipping incompatible /opt/docker/aufs/diff/1458d7dbf760601e9276866f8d31c399bb56268e8bf931a847893e315a749abb/pdt-toolchains/8/amd64/sysroot/usr/lib/i386-linux-gnu/libpcap.so when searching for -lpcap
/usr/bin/ld: skipping incompatible /opt/docker/aufs/diff/1458d7dbf760601e9276866f8d31c399bb56268e8bf931a847893e315a749abb/pdt-toolchains/8/amd64/sysroot/usr/lib/i386-linux-gnu/libpcap.a when searching for -lpcap
/usr/bin/ld: cannot find -lpcap
/usr/bin/ld: skipping incompatible /opt/docker/aufs/diff/1458d7dbf760601e9276866f8d31c399bb56268e8bf931a847893e315a749abb/pdt-toolchains/8/amd64/sysroot/usr/lib/i386-linux-gnu/libnsl.so when searching for -lnsl
/usr/bin/ld: skipping incompatible /opt/docker/aufs/diff/1458d7dbf760601e9276866f8d31c399bb56268e8bf931a847893e315a749abb/pdt-toolchains/8/amd64/sysroot/usr/lib/i386-linux-gnu/libnsl.a when searching for -lnsl
/usr/bin/ld: skipping incompatible /opt/docker/aufs/diff/1458d7dbf760601e9276866f8d31c399bb56268e8bf931a847893e315a749abb/pdt-toolchains/8/amd64/sysroot/usr/lib/i386-linux-gnu/libresolv.so when searching for -lresolv
/usr/bin/ld: skipping incompatible /opt/docker/aufs/diff/1458d7dbf760601e9276866f8d31c399bb56268e8bf931a847893e315a749abb/pdt-toolchains/8/amd64/sysroot/usr/lib/i386-linux-gnu/libresolv.a when searching for -lresolv
/usr/bin/ld: skipping incompatible /opt/docker/aufs/diff/1458d7dbf760601e9276866f8d31c399bb56268e8bf931a847893e315a749abb/pdt-toolchains/8/amd64/sysroot/usr/lib/i386-linux-gnu/libdl.so when searching for -ldl
/usr/bin/ld: skipping incompatible /opt/docker/aufs/diff/1458d7dbf760601e9276866f8d31c399bb56268e8bf931a847893e315a749abb/pdt-toolchains/8/amd64/sysroot/usr/lib/i386-linux-gnu/libdl.a when searching for -ldl
/usr/bin/ld: skipping incompatible /opt/docker/aufs/diff/1458d7dbf760601e9276866f8d31c399bb56268e8bf931a847893e315a749abb/pdt-toolchains/8/amd64/sysroot/usr/lib/i386-linux-gnu/libpthread.so when searching for -lpthread
/usr/bin/ld: skipping incompatible /opt/docker/aufs/diff/1458d7dbf760601e9276866f8d31c399bb56268e8bf931a847893e315a749abb/pdt-toolchains/8/amd64/sysroot/usr/lib/i386-linux-gnu/libpthread.a when searching for -lpthread
collect2: error: ld returned 1 exit status

**Comment by NolanWoo** [comment]
Is that helpful?

**Comment by arr2036** [comment]
Yes, pushed a fix, please try v3.0.x HEAD

**Comment by NolanWoo** [comment]
yes, it fixed the old problem. But I meet a new one, maybe my environment has some problem?
report:
In file included from src/lib/pcap.c:27:0:
src/freeradius-devel/pcap.h:72:16: error: ‘PCAP_ERRBUF_SIZE’ undeclared here (not in a function)
  char   errbuf[PCAP_ERRBUF_SIZE]; //!< Last error on this interface.
                ^
src/freeradius-devel/pcap.h:81:2: error: unknown type name ‘pcap_t’
  pcap_t   *handle;   //!< libpcap handle.
  ^
src/freeradius-devel/pcap.h:82:2: error: unknown type name ‘pcap_dumper_t’
  pcap_dumper_t  *dumper;   //!< libpcap dumper handle.
  ^
src/freeradius-devel/pcap.h:87:19: error: field ‘pstats’ has incomplete type
  struct pcap_stat pstats;    //!< The last set of pcap stats for this handle.
                   ^
src/freeradius-devel/pcap.h:92:43: error: unknown type name ‘pcap_if_t’
 int  fr_pcap_if_link_layer(char *errbuff, pcap_if_t *dev);
                                           ^
src/lib/pcap.c: In function ‘_free_pcap’:
src/lib/pcap.c:52:4: warning: implicit declaration of function ‘pcap_close’ [-Wimplicit-function-declaration]
    pcap_close(pcap->handle);
    ^
src/lib/pcap.c:64:4: warning: implicit declaration of function ‘pcap_dump_flush’ [-Wimplicit-function-declaration]
    pcap_dump_flush(pcap->dumper);
    ^
src/lib/pcap.c:65:4: warning: implicit declaration of function ‘pcap_dump_close’ [-Wimplicit-function-declaration]
    pcap_dump_close(pcap->dumper);
    ^
src/lib/pcap.c: At top level:
src/lib/pcap.c:86:42: error: unknown type name ‘pcap_if_t’
 int fr_pcap_if_link_layer(char *errbuff, pcap_if_t *dev)
                                          ^
src/lib/pcap.c: In function ‘fr_pcap_open’:
src/lib/pcap.c:135:3: warning: implicit declaration of function ‘pcap_create’ [-Wimplicit-function-declaration]
   pcap->handle = pcap_create(pcap->name, pcap->errbuf);
   ^
src/lib/pcap.c:140:3: warning: implicit declaration of function ‘pcap_set_snaplen’ [-Wimplicit-function-declaration]
   if (pcap_set_snaplen(pcap->handle, SNAPLEN) != 0) {
   ^
src/lib/pcap.c:142:4: warning: implicit declaration of function ‘pcap_geterr’ [-Wimplicit-function-declaration]
    fr_strerror_printf("%s", pcap_geterr(pcap->handle));
    ^
src/lib/pcap.c:142:4: warning: format ‘%s’ expects argument of type ‘char *’, but argument 2 has type ‘int’ [-Wformat=]
src/lib/pcap.c:147:3: warning: implicit declaration of function ‘pcap_set_timeout’ [-Wimplicit-function-declaration]
   if (pcap_set_timeout(pcap->handle, PCAP_NONBLOCK_TIMEOUT) != 0) {
   ^
src/lib/pcap.c:150:3: warning: implicit declaration of function ‘pcap_set_promisc’ [-Wimplicit-function-declaration]
   if (pcap_set_promisc(pcap->handle, pcap->promiscuous) != 0) {
   ^
src/lib/pcap.c:154:3: warning: implicit declaration of function ‘pcap_set_buffer_size’ [-Wimplicit-function-declaration]
   if (pcap_set_buffer_size(pcap->handle, SNAPLEN *
   ^
src/lib/pcap.c:158:3: warning: implicit declaration of function ‘pcap_activate’ [-Wimplicit-function-declaration]
   if (pcap_activate(pcap->handle) != 0) {
   ^
src/lib/pcap.c:176:3: warning: implicit declaration of function ‘pcap_setnonblock’ [-Wimplicit-function-declaration]
   if (pcap_setnonblock(pcap->handle, true, pcap->errbuf) != 0) {
   ^
src/lib/pcap.c:184:3: warning: implicit declaration of function ‘pcap_get_selectable_fd’ [-Wimplicit-function-declaration]
   pcap->fd = pcap_get_selectable_fd(pcap->handle);
   ^
src/lib/pcap.c:185:3: warning: implicit declaration of function ‘pcap_datalink’ [-Wimplicit-function-declaration]
   pcap->link_layer = pcap_datalink(pcap->handle);
   ^
src/lib/pcap.c:198:3: warning: implicit declaration of function ‘pcap_open_offline’ [-Wimplicit-function-declaration]
   pcap->handle = pcap_open_offline(pcap->name, pcap->errbuf);
   ^
src/lib/pcap.c:210:23: error: ‘DLT_EN10MB’ undeclared (first use in this function)
    pcap->link_layer = DLT_EN10MB;
                       ^
src/lib/pcap.c:210:23: note: each undeclared identifier is reported only once for each function it appears in
src/lib/pcap.c:212:3: warning: implicit declaration of function ‘pcap_open_dead’ [-Wimplicit-function-declaration]
   pcap->handle = pcap_open_dead(pcap->link_layer, SNAPLEN);
   ^
src/lib/pcap.c:212:16: warning: assignment makes pointer from integer without a cast [enabled by default]
   pcap->handle = pcap_open_dead(pcap->link_layer, SNAPLEN);
                ^
src/lib/pcap.c:218:3: warning: implicit declaration of function ‘pcap_dump_open’ [-Wimplicit-function-declaration]
   pcap->dumper = pcap_dump_open(pcap->handle, pcap->name);
   ^
src/lib/pcap.c:218:16: warning: assignment makes pointer from integer without a cast [enabled by default]
   pcap->dumper = pcap_dump_open(pcap->handle, pcap->name);
                ^
src/lib/pcap.c:220:4: warning: format ‘%s’ expects argument of type ‘char *’, but argument 2 has type ‘int’ [-Wformat=]
    fr_strerror_printf("%s", pcap_geterr(pcap->handle));
    ^
src/lib/pcap.c:228:3: warning: implicit declaration of function ‘pcap_fopen_offline’ [-Wimplicit-function-declaration]
   pcap->handle = pcap_fopen_offline(stdin, pcap->errbuf);
   ^
src/lib/pcap.c:246:3: warning: implicit declaration of function ‘pcap_dump_fopen’ [-Wimplicit-function-declaration]
   pcap->dumper = pcap_dump_fopen(pcap->handle, stdout);
   ^
src/lib/pcap.c:246:16: warning: assignment makes pointer from integer without a cast [enabled by default]
   pcap->dumper = pcap_dump_fopen(pcap->handle, stdout);
                ^
src/lib/pcap.c:248:4: warning: format ‘%s’ expects argument of type ‘char *’, but argument 2 has type ‘int’ [-Wformat=]
    fr_strerror_printf("%s", pcap_geterr(pcap->handle));
    ^
src/lib/pcap.c: In function ‘fr_pcap_apply_filter’:
src/lib/pcap.c:277:2: error: unknown type name ‘bpf_u_int32’
  bpf_u_int32 mask = 0;    /* Our netmask */
  ^
src/lib/pcap.c:278:2: error: unknown type name ‘bpf_u_int32’
  bpf_u_int32 net = 0;    /* Our IP */
  ^
src/lib/pcap.c:279:21: error: storage size of ‘fp’ isn’t known
  struct bpf_program fp;
                     ^
src/lib/pcap.c:298:3: warning: implicit declaration of function ‘pcap_lookupnet’ [-Wimplicit-function-declaration]
   if (pcap_lookupnet(pcap->name, &net, &mask, pcap->errbuf) < 0) {
   ^
src/lib/pcap.c:304:2: warning: implicit declaration of function ‘pcap_compile’ [-Wimplicit-function-declaration]
  if (pcap_compile(pcap->handle, &fp, expression, 0, net) < 0) {
  ^
src/lib/pcap.c:305:3: warning: format ‘%s’ expects argument of type ‘char *’, but argument 2 has type ‘int’ [-Wformat=]
   fr_strerror_printf("%s", pcap_geterr(pcap->handle));
   ^
src/lib/pcap.c:310:2: warning: implicit declaration of function ‘pcap_setfilter’ [-Wimplicit-function-declaration]
  if (pcap_setfilter(pcap->handle, &fp) < 0) {
  ^
src/lib/pcap.c:311:3: warning: format ‘%s’ expects argument of type ‘char *’, but argument 2 has type ‘int’ [-Wformat=]
   fr_strerror_printf("%s", pcap_geterr(pcap->handle));
   ^
src/lib/pcap.c:279:21: warning: unused variable ‘fp’ [-Wunused-variable]
  struct bpf_program fp;
                     ^
src/lib/pcap.c: In function ‘fr_pcap_link_layer_supported’:
src/lib/pcap.c:362:7: error: ‘DLT_EN10MB’ undeclared (first use in this function)
  case DLT_EN10MB:
       ^
src/lib/pcap.c:363:7: error: ‘DLT_RAW’ undeclared (first use in this function)
  case DLT_RAW:
       ^
src/lib/pcap.c:364:7: error: ‘DLT_NULL’ undeclared (first use in this function)
  case DLT_NULL:
       ^
src/lib/pcap.c:365:7: error: ‘DLT_LOOP’ undeclared (first use in this function)
  case DLT_LOOP:
       ^
src/lib/pcap.c:369:7: error: ‘DLT_PFLOG’ undeclared (first use in this function)
  case DLT_PFLOG:
       ^
src/lib/pcap.c: In function ‘fr_pcap_link_layer_offset’:
src/lib/pcap.c:398:7: error: ‘DLT_RAW’ undeclared (first use in this function)
  case DLT_RAW:
       ^
src/lib/pcap.c:401:7: error: ‘DLT_NULL’ undeclared (first use in this function)
  case DLT_NULL:
       ^
src/lib/pcap.c:402:7: error: ‘DLT_LOOP’ undeclared (first use in this function)
  case DLT_LOOP:
       ^
src/lib/pcap.c:412:7: error: ‘DLT_EN10MB’ undeclared (first use in this function)
  case DLT_EN10MB:
       ^
src/lib/pcap.c:460:7: error: ‘DLT_PFLOG’ undeclared (first use in this function)
  case DLT_PFLOG:
       ^
src/lib/pcap.c: At top level:
cc1: warning: unrecognized command line option "-Wno-date-time" [enabled by default]
make: *** [build/objs/src/lib/pcap.lo] Error 1
